src/python/grpcio_channelz/grpc_channelz/v1/_servicer.py

```diff
@@ -24,7 +26,12 @@ class ChannelzServicer(_channelz_pb2_grpc.ChannelzServicer):
     """Servicer handling RPCs for service statuses."""
 
     @staticmethod
-    def GetTopChannels(request, context):
+    def GetTopChannels(
+        request: _channelz_pb2.GetTopChannelsRequest,
+        context: Union[
```

Comment:
I expect to see this `Union` type show up in many places. Should we create an alias type and call it something like `grpc.GenericServicerContext`?

Reply:
moved this to top as single const and using it in entire file

Reply:
Moved to grpc __init__.py using from there